Makefile

```diff
@@ -13,6 +14,8 @@ build-extension: ## Build service image to be deployed as a desktop extension
 		--secret id=BUGSNAG_API_KEY \
 		--build-arg BUGSNAG_RELEASE_STAGE=$(BUGSNAG_RELEASE_STAGE) \
 		--build-arg BUGSNAG_APP_VERSION=$(TAG) \
+		--label org.opencontainers.image.source=https://github.com/docker/volume-backup-extension \
+		--label org.opencontainers.image.revision=$(GIT_COMMIT) \
```

Comment:
Apparently, the `GIT_COMMIT` doesn't work in PRs as it leaves the CI in a waiting state forever: 

`docker-vulnerability-policy Expected — Waiting for status to be reported`.

We need to keep `${{ github.event.pull_request.head.sha || github.event.after || github.event.release.tag_name }}` as the value for the `org.opencontainers.image.revision` label. 

I'll add a commit with the fix.